codec-agnostic-rtp-payload-format.md

```diff
@@ -43,27 +43,37 @@ informative:
   RFC6464:
   RFC6465:
   RFC6904:
+  RFC6184:
   SFrame:
+    target: https://tools.ietf.org/html/draft-omara-sframe
+    title: Secure Frame (SFrame)
+  WebRTCInsertableStreams:
+    target: https://w3c.github.io/webrtc-insertable-streams
+    title: WebRTC Insertable Media using Streams
 
 --- abstract
 
-
+WebRTC Media Chains usually rely on piping encoder output directly to packetizers. Media packetization formats often support a specific codec format and optimize RTP packets generation accordingly.
+With the development of Selective Forward Unit (SFU) solutions, WebRTC Media Chains are increasingly relying on application-specific transforms that seat between encoder and packetizer on one end and between depacketizer and decoder on the other end. These transforms are typically protecting media content from being accessible from the SFU, for instance using {{SFrame}} or {{WebRTCInsertableStreams}}.
+In that context, RTP packetizers can no longer expect to use packetization formats that mandate media content to be in a specific codec format.
```

Comment:
Same, imho is better to focus on the SFrame/media encryption case stating that current codec-aware packetizer doesn't work with encrypted content and a generic packetizer is required instead

Reply:
Ditto here: protecting media content from being accessible from the SFU means double encryption. Not sure how to be clearer.